build.sh

```diff
@@ -1,5 +1,30 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+# This script is a modification of the build for docke2aci
+# https://github.com/appc/docker2aci/blob/master/build.sh
+
+set -e
+
+ORG_PATH="github.com/kinvolk"
+REPO_PATH="${ORG_PATH}/tcd"
+
+if [ ! -h gopath/src/${REPO_PATH} ]; then
+    mkdir -p gopath/src/${ORG_PATH}
+    ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
+fi
+
+export GOBIN=${PWD}/bin
+export GOPATH=${PWD}/gopath:${PWD}/Godeps/_workspace
+
+eval $(go env)
+go get -d
+
+echo "Building tcd..."
+CGO_ENABLE=0 go build -o $GOBIN/tcd -installsuffix cgo
+
+
 
 # ~/programs/protoc/protoc -I api api/service.proto --go_out=plugins=grpc:api
-CGO_ENABLED=0 go build -o bin/tcd -installsuffix cgo
+#CGO_ENABLED=0 go build -o bin/tcd -installsuffix cgo
```

Comment:
No need to keep the "go build" line commented, since you have the new version above.